source/2020-01-31-the-ember-times-issue-133.md

```diff
@@ -68,6 +69,29 @@ If you would like to learn more or add your feedback please take a look at the [
 
 ---
 
+## [Confident contributing to open-source at EmberConf 💪💻](https://emberconf.com/#/speakers/anne-greeth-van-herwijnen)
+
+After her [previous OSS pitch at Emberfest 2019](https://www.youtube.com/watch?v=GHbjdesHhPA),
+Emberista and [Ember Times](https://blog.emberjs.com/tags/newsletter.html) editor [Anne-Greeth van Herwijnen (@MinThaMie)](https://github.com/MinThaMie) will guide you to your next (or maybe even first!) open-source contribution!
+
+With plenty of research on what makes an open-source community successful in facilitating contributions, she will demonstrate how Ember empowers contributors to give back to the project in her talk [**Why Contributing Seems Scary**](https://emberconf.com/#/schedule/day-2_why-contributing-seems-scary) at [this year's EmberConf](https://emberconf.com/).
+
+In an exclusive interview with The Ember Times, Anne-Greeth shares with us, who she thinks is qualified to contribute to Ember:
+
+> […] I think everyone can contribute to Ember. It just matters on your opinion of what you think contributing means. So for me contributing goes from helping out at EmberConf, to writing, reading RFCs, and everything in between. And I think as long as you want to. you can always find something for you to contribute to.
+
+What is something that surprises first-time contributors when working on a project like Ember? Anne-Greeth speaks from her previous experience when sharing her OSS journey with others:
+
+> From the talk about contributing I gave at EmberFest last year and also one I gave at my work, I noticed that people were surprised how easy it is to contribute. And that contributing doesn’t mean that you necessarily have to fix the problem, but that also identifying a problem is contributing to it. At my company I got the response from somebody who just two hours after my talk already created their first pull request to an open-source repository (which wasn’t Ember), but still he was like “Hey, I did this!”  In general, the feedback has been really positive, because my examples are really low-key. People feel really encouraged and inspired, that they can do it, too!
```

Comment:
```suggestion
<!--alex ignore just easy-->
> From the talk about contributing I gave at EmberFest last year and also one I gave at my work, I noticed that people were surprised how easy it is to contribute. And that contributing doesn’t mean that you necessarily have to fix the problem, but that also identifying a problem is contributing to it. At my company I got the response from somebody who just two hours after my talk already created their first pull request to an open-source repository (which wasn’t Ember), but still he was like “Hey, I did this!”  In general, the feedback has been really positive, because my examples are really low-key. People feel really encouraged and inspired, that they can do it, too!
```